dids/src/test/kotlin/web5/sdk/dids/methods/dht/DidDhtTest.kt

```diff
@@ -164,6 +164,20 @@ class DidDhtTest {
       }
     }
 
+    @Test
```

Comment:
I included this to prove it passes. I'll remove it and push again after CI succeeds

Reply:
update: I captured the known hex response so this test can work without relying on the DID being published